extensions/rbac/verify.go

```diff
@@ -34,6 +36,8 @@ const (
 	standardUser                  = "user"
 	activeStatus                  = "active"
 	forbiddenError                = "403 Forbidden"
+	localcluster                  = "local"
```

Comment:
nit., Can you change the variable name to `localCluster`?

Reply:
changed constant name 👍